pkgs/servers/rtsp-simple-server/default.nix

```diff
@@ -5,22 +5,22 @@
 
 buildGoModule rec {
   pname = "rtsp-simple-server";
-  version = "0.15.4";
+  version = "0.17.1";
 
   src = fetchFromGitHub {
     owner = "aler9";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-6XdX4HEjDRt9WtqyHIv/NLt7IytNDeJLgCeTHTGybRI=";
+    sha256 = "sha256-8g9taSFEprJEEPM0hrbCf5QDE41uVdgVVIWjhfEICpU=";
   };
 
-  vendorSha256 = "sha256-T5LWbxYsKnG5eaYLR/rms6+2DXv2lV9o39BvF7HapZY=";
+  vendorSha256 = "sha256-buQW5jMnHyHc/oYdmfTnoktFRG3V3SNxn7t5mAwmiJI=";
 
   # Tests need docker
   doCheck = false;
 
   buildFlagsArray = [
-    "-ldflags=-X main.Version=${version}"
```

Comment:
Are you sure this does not work? Upstream seems to be still using this:

https://github.com/aler9/rtsp-simple-server/blob/d7f1c04711a995459c6ec6f5930d8b3cbdc99fdd/Makefile#L233

Reply:
Yep! This looks like a bug upstream, actually - I'll be filing an issue at some point. You can see in the `main.go` file that there's no `version` variable:
https://github.com/aler9/rtsp-simple-server/blob/v0.17.1/main.go

And that there is one in `internal/core`:
https://github.com/aler9/rtsp-simple-server/blob/v0.17.1/internal/core/core.go#L18

Which is used here:
https://github.com/aler9/rtsp-simple-server/blob/v0.17.1/internal/core/core.go#L56-L59